client/context.go

```diff
@@ -333,13 +333,19 @@ func (ctx Context) printOutput(out []byte) error {
 // GetFromFields returns a from account address, account name and keyring type, given either
 // an address or key name. If genOnly is true, only a valid Bech32 cosmos
 // address is returned.
-func GetFromFields(kr keyring.Keyring, from string, genOnly bool) (sdk.AccAddress, string, keyring.KeyType, error) {
+func GetFromFields(kr keyring.Keyring, from string, genOnly bool, simulate bool) (sdk.AccAddress, string, keyring.KeyType, error) {
```

Comment:
Curious why we don't use the same fix as in 0.46 where we pass `clientCtx Context`. Then we can get the GenerateOnly and Simulate booleans.

Reply:
Its true, but we can avoid passing heavy context object since we aren't using any other fields other than these two here.

Reply:
Right, but it is an unnecessary API breaking change between 0.45 and 0.46 then. People that upgrade from 0.45 to 0.46 will need to change twice: e.g https://github.com/desmos-labs/desmos/blob/dd2162e/app/desmos/cmd/sign/sign.go#L36-L40

Reply:
I would rather pass in `Context` here. It future-proofs against additional requirements.